### PR TITLE
EDU-52 Make translations overridable

### PR DESCRIPTION
**Pull request by ahelmberger**

### CODE REVIEW

File: src/resources/resource-manager.js [hunk 1]
```diff
@@ -1,13 +1,30 @@
 import i18next from 'i18next';
 import icu from 'i18next-icu';
+import cloneDeep from '../utils/clone-deep.js';
 import { initReactI18next } from 'react-i18next';
 import { SUPPORTED_UI_LANGUAGES } from './ui-language.js';
 
 const DEFAULT_NAMESPACE = 'default';
 
 class ResourceManager {
```
Comment (laurentiu-ilici) [inline src/resources/resource-manager.js:9]: DbManager... :)))

**Review by laurentiu-ilici — APPROVED** [review]

File: src/resources/resource-manager.js [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
-  constructor(resourceBundles) {
-    this._resourceBundles = resourceBundles;
+  constructor(...resourceBundles) {
+    this._resourceBundles = this._mergeResourceBundles(resourceBundles);
+  }
+
+  _mergeResourceBundles(bundles) {
+    const result = [];
+    for (const bundle of bundles) {
+      for (const entry of bundle) {
```
Comment (adrianaluchian) [inline src/resources/resource-manager.js:17]: I think you could do `bundles.flat().forEach(bundle => { .. })`, or?
And small thing, could `existing` become `existingBundle` please?

File: src/resources/resource-manager.js (hunk 1 above)
Comment (ahelmberger) [reply src/resources/resource-manager.js:9]: hehe :-)

File: src/resources/resource-manager.js (hunk 2 above)
Comment (ahelmberger) [reply src/resources/resource-manager.js]: Yup, good idea, both done (although I named it `existingEntry`)!

**Review by adrianaluchian — APPROVED** [review]

**Pull request merged**